keyform/templates/keyform/searchtable.html

```diff
@@ -0,0 +1,66 @@
+{% load i18n %}
+<div class="row">
+    <div class="large-12 columns">
+        <button id="search-button">{% trans 'Search' %}</button>
+        <br>
+    </div>
+</div>
+<div class="search-form" id="search">
+    <div class="search-content">
+        <form action="{% url 'search' %}">
+            <fieldset>
+                <div class="row">
+                    <div class="large-4 columns">
+                       {% trans 'Student Name' %} <input type="text" name="student_name"/>
```

Comment:
On the search results page, set the value of this field (and all the others) to what was searched by